tests/cases/fourslash/refactorKind_rewriteFunction.ts

```diff
@@ -7,6 +7,5 @@ verify.refactorKindAvailable("refactor.rewrite",
 [
     "refactor.rewrite.arrow.braces.add",
     "refactor.rewrite.function.named",
-    "refactor.rewrite.function.anonymous",
-    "refactor.rewrite.function.returnType"

```

Comment:
Can you add another example in this test that will work for `"refactor.rewrite.function.returnType"`? This is here to make sure that requesting this refactor by name gives us what we want.

Reply:
@jessetrinity Changed code sample to have `"refactor.rewrite.function.returnType"`.